packages/loot-core/src/server/accounts/sync.ts

```diff
@@ -116,19 +111,31 @@ async function downloadGoCardlessTransactions(
     throw BankSyncError(res.error_type, res.error_code);
   }
 
-  const {
-    transactions: { all },
-    balances,
-    startingBalance,
-  } = res;
+  if (includeBalance) {
+    const {
+      transactions: { all },
+      balances,
+      startingBalance,
+    } = res;
```

Comment:
Similar question as below, why not just build what we want to return and then return `res`?

Reply:
I prefer it in this case to the alternative:

```js
    return {                                                                    
      transactions: res.transactions.all,                                       
      accountBalance: res.balances,                                             
      startingBalance: res.startingBalance,                                     
    };
```

Reply:
We also want to log the original `res` out for debugging purposes